dist/index.html

```diff
@@ -206,52 +196,63 @@ <h5 class="gallery__project-type">
           </picture>
         </li>
       </ul>
-      <div class="gallery__buttons">
-        <ul class="gallery__btn-container">
-          <button class="gallery__btn gallery__btn1" id="gallery__btn1">
-            <a href="#gallery__element1"></a>
-          </button>
-          <button class="gallery__btn gallery__btn2" id="gallery__btn2">
-            <a href="#gallery__element2"></a>
-          </button>
-          <button class="gallery__btn gallery__btn3" id="gallery__btn3">
-            <a href="#gallery__element3"></a>
-          </button>
-        </ul>
-      </div>
     </section>
 
     <!-- Contact form -->
     <section class="contact" id="contact">
+      <div class="contact__sent-info--disabled">
+        <ion-icon
+          class="contact__sent-info__icon"
+          name="close-outline"
+        ></ion-icon>
+        <span class="contact__sent-info__p">Wiadomość została wysłana</span>
+      </div>
       <h6 class="contact__heading">Napisz do mnie</h6>
-      <form class="contact__form">
+
+      <form
+        action="https://formsubmit.co/test.page0987@gmail.com"
+        class="contact__form"
+        method="post"
+        action="send"
+        enctype="multipart/form-data"
+      >
         <input
-          class="contact__email"
+          class="contact__email hover"
           type="email"
+          name="email"
           placeholder="wpisz swój adres email"
+          required
         />
         <p class="contact__email-info"></p>
         <textarea
-          class="contact__msg"
+          class="contact__msg hover"
           name="message"
           rows="4"
           cols="50"
           placeholder="wpisz treść wiadomości"
+          required
         ></textarea>
-        <button class="contact__btn">Wyślij</button>
+        <input
+          type="hidden"
+          name="_next"
+          value="http://localhost:3000/after_msg.html"
```

Comment:
localhost:3000 to lokalny serwer - tutaj powinna być ścieżka absolutna `./after_msg.html`